specs/094-callout-publisher-meta/research.md

```
@@ -0,0 +1,91 @@
+- **Decision**: Each of the three callout mappers in `calloutDataMapper.ts` (`mapCalloutLightToPostCard`, `mapCalloutDetailsToPostCard`, `mapCalloutDetailsToDialogData`) prepends a two-line preamble:
+  ```ts
+  const authorSource = callout.publishedBy ?? callout.createdBy;
+  const dateSource = callout.publishedDate ?? callout.createdDate;
+  ```
+  then references `authorSource` and `dateSource` in the existing `author` / `timestamp` expressions. The search mapper in `searchDataMapper.ts` (one call site, `mappedCallouts`) follows the same pattern, with one extra step: the resolved name is then passed through the existing `?? unknownLabel` fallback that already exists for whiteboard/memo/post results, so a callout with neither publisher nor creator still renders the localised "Unknown" label.
+- **Rationale**:
+  - Mirrors the MUI implementation (`author={callout.publishedBy ?? callout.createdBy}`) — same conceptual operation, applied at each layer's "boundary point."
+  - Keeps each mapper self-contained and readable; the source of truth for each field is one line away from where it's used.
+  - For search specifically, layering the existing `?? unknownLabel` on top of the `??` chain preserves consistency with how whiteboard/memo/post search results already render and avoids introducing a new "no author" branch.
+- **Alternatives considered**:
+  - *Inline `?? ` directly in the `author: { name: …, avatarUrl: …}` literal*: rejected — produces an unreadably nested expression; the local `const` approach is one line longer per mapper but markedly clearer.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)